doc/python/creating-and-updating-figures.md

```diff
@@ -271,31 +270,20 @@ Graph object figures support an `update_layout` method that may be used to updat
 
 ```python
 import plotly.graph_objects as go
+
 fig = go.Figure(data=go.Bar(x=[1, 2, 3], y=[1, 3, 2]))
-fig.update_layout(title_text="A Bar Chart",
-                  title_font_size=30)
+fig.update_layout(title_text="A Bar Chart", title_font_size=30)
 fig.show()
 ```
 
 Note that the following `update_layout` operations are equivalent:
 
 ```python
-fig.update_layout(title_text="A Bar Chart",
-                  title_font_size=30)
-
-fig.update_layout(title_text="A Bar Chart",
-                  title_font=dict(size=30))
-
-
-fig.update_layout(title=dict(text="A Bar Chart"),
-                             font=dict(size=30))
-
-fig.update_layout({"title": {"text": "A Bar Chart",
-                             "font": {"size": 30}}})
-
-fig.update_layout(
-    title=go.layout.Title(text="A Bar Chart",
-                          font=go.layout.title.Font(size=30)));
+fig.update_layout(title_text="A Bar Chart",title_font_size=30)
+fig.update_layout(title_text="A Bar Chart", title_font=dict(size=30))
+fig.update_layout(title=dict(text="A Bar Chart"), font=dict(size=30))
+fig.update_layout({"title": {"text": "A Bar Chart", "font": {"size": 30}}})
+fig.update_layout(title=dict(text="A Bar Chart", font=dict(size=30)))
```

Comment:
I think here we want to keep the `go.layout.Title` because we're showing equivalent syntaxes, although the recommended one does not use `go.layout`.